app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -66,6 +66,8 @@ class PatientSummaryScreenController @Inject constructor(
     val replayedEvents = ReplayUntilScreenIsDestroyed(events)
         .compose(ReportAnalyticsEvents())
         .compose(mergeWithPatientSummaryChanges())
+        .compose(mergeWithBloodPressureSaves())
+        .compose(mergeWithAllBloodPressuresDeleted())
```

Comment:
Yes, to me this indicates that the the transformer is going to add new events to the stream. Otherwise, I assume it's going to transform the stream and drop some of the source events in some fashion. Does that make sense?

Reply:
Agreed. But we should follow this convention elsewhere too. We'll now have a mix of multiple conventions and that's not good.

Reply:
True. In this case, I basically just followed the same convention which we had with `mergeWithPatientSummaryChanges`. `mergeWithBloodPressureSaves` has been deleted in a later commit. I can rename `mergeWithAllBloodPressuresDeleted` to something like `checkWhetherAllBloodPressuresAreDeleted` or something similar.

For what it's worth, I checked for usages of the `.compose` method on `ReplayUntilScreenIsDestroyed`, and I see that we are using a mix of "mergeWith" prefixed and non-prefixed throughout the codebase.